plugins/Marketplace/templates/getPremiumFeatures.twig

```diff
@@ -1,6 +1,6 @@
 <div class="getNewPlugins getPremiumFeatures widgetBody">
     <div class="row">
-        {% for plugin in plugins %}
+        {% for plugin in plugins if not plugin.isBundle %}
```

Comment:
Is there a special case preventing you from using the same `array_filter` like the plugin listing? Filtering the entries here seems to add an unintuitive extra limit to the already max 20 entries.

Reply:
good point, will update PR

Reply:
Updated :)